src/containers/LandingPage/LandingPage.css

```diff
@@ -85,13 +85,19 @@
   margin: var(--LandingPage_sectionMarginTop) 24px 51px 24px;
 
   @media (--viewportMedium) {
-    margin: var(--LandingPage_sectionMarginTopMedium) 7.5vw 60px 7.5vw;
+    max-width: 100%;
+    margin: var(--LandingPage_sectionMarginTopMedium) 30px 60px 30px;
```

Comment:
I was a bit worried about what's going on here (since Topbar uses 24px horizontal paddings), but I guess, we need to ask from Janne: `(Janne Koivistoinen 2017-12-22 15:33:19 +0200 13)     padding: 0 30px 33px 30px;`